src/ChangeStream.php

```diff
@@ -197,7 +197,7 @@ public function valid()
      * @param RuntimeException $exception
      * @return boolean
      */
-    private function isResumableError(RuntimeException $exception)
+    private function isResumableError(RuntimeException $exception): bool
```

Comment:
Does it make sense to remove redundant doc block `@param` and `@return` lines that don't add any value? I think we'd just leave them in place if particular parameters have associated comments.

Reply:
I've enabled the `UselessAnnotation` checks for parameter, property, and return types. `phpcbf` removed a number of useless annotations and will ensure these are no longer added.